docs/releases/3.1.txt

```diff
@@ -96,6 +96,10 @@ Minor features
 * The password reset mechanism now uses the SHA-256 hashing algorithm. Support
   for tokens that use the old hashing algorithm remains until Django 4.0.
 
+* The :func:`~django.contrib.auth.hashers.make_password` now requires its
```

Comment:
This goes under "backward-incompatible changes", not "minor features". Probably the rationale is worth mentioning as well.